website/src/css/custom.css

```diff
@@ -442,62 +699,25 @@ nav.navbar div.navbar__inner {
   background: var(--color-primary-blue);
 }
 
-@media (min-width: 997px) and (max-width: 1150px) {
-  nav.navbar a.navbar__link {
-    font-size: 0.9rem;
-  }
-  .navbar__search .navbar__search-input {
-    max-width: 160px;
-  }
-}
-
-@media screen and (max-width: 700px) {
-  .card.large {
-    height: auto;
-    text-align: center;
-  }
-  .container.container--fluid.home .row {
-    margin: calc(2vh) auto !important;
-  }
-}
-
-/* V2.0.0-beta.6 style updates */
-.padding-top--md {
-  padding-top: 2rem !important;
-}
-
-code {
-  border: none;
-  background-color: var(--ifm-color-emphasis-200);
-  color: var(--ifm-color-emphasis-900);
-}
-
-.DocSearch-Button-Container .DocSearch-Button-Placeholder {
-  padding: 0 2.25rem 0 0;
-}
-
-@media (min-width: 1440px) {
-  .container--fluid {
-    max-width: inherit;
-  }
-}
-
 /* Custom Blog Styles */
 .blog-breadcrumbs {
   display: block;
   margin-bottom: 0;
   background: var(--color-off-white);
   padding: .5rem 0;
 }
```

Comment:
The blog breadcrumb styles do not update when switching to dark mode.

![image](https://user-images.githubusercontent.com/13647110/190256734-5fc0f07a-d7b9-4810-9355-d4abad0f6569.png)

Reply:
breadcrumbs should now look good on dark mode:
https://deploy-preview-1990--docs-getdbt-com.netlify.app/docs/introduction